js/src/playground/playground.js

```diff
@@ -0,0 +1,88 @@
+// Copyright 2015, 2016 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import { observer } from 'mobx-react';
+import React, { Component } from 'react';
+
+import PlaygroundStore from './store';
+import styles from './playground.css';
+
+import CurrencySymbol from '~/ui/CurrencySymbol/currencySymbol.example';
+import QrCode from '~/ui/QrCode/qrCode.example';
+import SectionList from '~/ui/SectionList/sectionList.example';
+
+PlaygroundStore.register(<CurrencySymbol />);
+PlaygroundStore.register(<QrCode />);
+PlaygroundStore.register(<SectionList />);
+
+@observer
+export default class Playground extends Component {
+  state = {
+    selectedIndex: 0
+  };
+
+  store = PlaygroundStore.get();
+
+  render () {
```

Comment:
I would add at least a smoketest for any new components (i.e. it does render). While this is not meant for production, good habit to form and even the basic test does pick up some issues.